docs/configuration-reference/components/cert-manager.md

```diff
@@ -23,7 +23,6 @@ cert-manager component configuration example:
 component "cert-manager" {
```

Comment:
Nit: (being pedantic) the commit message is not wraped to 72 chars. The PR is good :+1: .

Reply:
True. I can recommend configuring your editor to wrap the lines for you for Git commit messages. I think `vim` does that automagically.

Reply:
Changed my editor settings :) fixed now!